xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java

```diff
@@ -151,7 +157,30 @@ public String getSection()
     {
         return this.section;
     }
-
+    
+    /**
+     * @param excludeHeading {@code true} to remove the first heading found inside
+     *         the document, {@code false} to keep it 
+     * @since 12.3RC1 
+     */
+    @Unstable
+    @PropertyName("Exclude Heading")
+    @PropertyDescription("Exclude the first heading from the included document/section.")
+    public void setExcludeHeading(boolean excludeHeading)
+    {
+        this.excludeHeading = excludeHeading;
+    }
+    
+    /**
+     * @return whether the user want to exclude the document/section headline.
```

Comment:
-> "@return whether the user wants to exclude the first heading from the included document or section"